src/wsdl/index.ts

```diff
@@ -98,7 +98,7 @@ export class WSDL {
 
         try {
           this.definitions.deleteFixedAttrs();
-          const services = this.services = this.definitions.services;
+          const services = (this.services = this.definitions.services);
```

Comment:
I find it a bit weird in some cases it add `(...)` around expressions and sometimes removes where I'd kept it for readability. But it's a minor thing.